### PR TITLE
Address more precise types from @types/node

### PR DESCRIPTION
**Pull request by vazexqi**
### What does this PR do?

Address the type changes due to https://github.com/DefinitelyTyped/DefinitelyTyped/commit/09ff4cfc0a0677e498f537640dfefa7fb40fbba1#diff-8811b50169fd98160ad78b073b4d0163

### What issues does this PR fix or reference?

@W-5086993@

### CODE REVIEW

**Comment by vazexqi** [comment]
@lcampos @DatGuyJonathan - Could you review this since it's a blocker for tomorrow's release? We should also look to see if we need to forward port this to develop first.

File: docs/localization.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ function loadMessageBundle(config?: Config): Message {
 }
 
 export const nls = new Localization(
-  loadMessageBundle(JSON.parse(process.env.VSCODE_NLS_CONFIG))
+  loadMessageBundle(process.env.VSCODE_NLS_CONGIF ? JSON.parse(process.env.VSCODE_NLS_CONFIG!) : undefined)
```
Comment (lcampos) [inline docs/localization.md]: @vazexqi think that ! at the end of JSON.parse is an issue.

**Comment by vazexqi** [comment]
sketch

Verified that even if you switch locales, nothing malfunctions. We don't have language files for any other language so the default behavior is to fall back on en.

File: docs/localization.md (hunk 1 above)
Comment (vazexqi) [reply docs/localization.md]: @lcampos - Sorry, why would that be an issue? The bang is the non-null assertion operator. See https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator

There is an issue with `process.env.VSCODE_NLS_CONGIF` though, which I will fix.

File: packages/salesforcedx-apex-debugger/src/messages/index.ts [hunk 2]
```diff
@@ -42,5 +42,5 @@ function loadMessageBundle(config?: Config): Message {
 }
 
 export const nls = new Localization(
-  loadMessageBundle(JSON.parse(process.env.VSCODE_NLS_CONFIG))
+  loadMessageBundle(process.env.VSCODE_NLS_CONGIF ? JSON.parse(process.env.VSCODE_NLS_CONFIG!) : undefined)
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-apex-debugger/src/messages/index.ts]: I think `CONGIF` needs to be `CONFIG`, here and other places as well

Comment (DatGuyJonathan) [reply packages/salesforcedx-apex-debugger/src/messages/index.ts]: Oops, yea, you caught it faster!

**Comment by DatGuyJonathan** [comment]
Verified that I can clean, install, compile, and package this branch, so I think I should be okay for the release.

**Comment by vazexqi** [comment]
The build is failing because of the SLDS issue that was fixed in #479 but not backported to this branch. I'm going to close this and also cherry-pick this into develop.

**Comment by vazexqi** [comment]
- [ ] Initial code review done 
- [ ] Changes based on code review
- [ ] Verify if this preference will also be picked up by the Apex Language Server for the code lens

**Pull request merged**